packages/plugin-core/src/services/EngineAPIService.ts

```diff
@@ -36,13 +37,19 @@ import {
   SchemaModuleDict,
   SchemaModuleProps,
 } from "@dendronhq/common-all";
-import { DendronEngineClient, HistoryService } from "@dendronhq/engine-server";
+import {
+  DendronEngineClient,
+  EngineEvents,
+  HistoryService,
+} from "@dendronhq/engine-server";
 import _ from "lodash";
 import { IEngineAPIService } from "./EngineAPIServiceInterface";
 
-export class EngineAPIService implements DEngineClient, IEngineAPIService {
-  private internalEngine: DEngineClient;
-
+export class EngineAPIService
```

Comment:
see comment in engineClient.ts about returning an emitter

Reply:
let's discuss on other comment thread in `[EngineEvents.spec.ts]`